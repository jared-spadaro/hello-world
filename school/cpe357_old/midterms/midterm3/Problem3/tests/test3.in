newList
addAtIndex 0 5.37
printList 1
checkSpace 1
addAtIndex 0 -6.31
printList 2
checkSpace 2
addAtIndex 2 9.73
printList 3
checkSpace 3
addAtIndex 1 3.54
printList 4
checkSpace 4
addAtIndex 3 8.82
printList 5
checkSpace 5
done
