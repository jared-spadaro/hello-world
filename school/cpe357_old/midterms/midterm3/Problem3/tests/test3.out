List: 5.37 
checkSpace: Expected amount of memory allocated
List: -6.31 5.37 
checkSpace: Expected amount of memory allocated
List: -6.31 5.37 9.73 
checkSpace: Expected amount of memory allocated
List: -6.31 3.54 5.37 9.73 
checkSpace: Expected amount of memory allocated
List: -6.31 3.54 5.37 8.82 9.73 
checkSpace: Expected amount of memory allocated
